.travis.yml

```diff
@@ -26,11 +26,18 @@ script:
 after_success:
  - codecov
 
+env:
+  global:
+    GIT_INSTALLER_DIR_PATH: ${HOME}/.git-installers
```

Comment:
Please add a comment that these "globla" vars are needed for macos only.

Reply:
It doesn't really matter. I can even move these to macOS only...

Reply:
Please do

Reply:
I just remembered why I made it global: this way we don't overwrite the values in individual jobs. I'll keep it here.